server/models/itineraryModel.js

```diff
@@ -1,22 +1,33 @@
-const mongoose = require('mongoose')
+/**
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ * RITHIN PLS USE THE TRIP AND STAGE MODEL
+ */
```

Comment:
Ya sure, but why is this merged in?

Reply:
Think we can delete the file if we are not using it anymore

Reply:
Fixed in new PR (because I wasn't sure if you still needed this as reference or smt)!